go/vt/worker/vertical_split_diff.go

```diff
@@ -63,11 +63,11 @@ type VerticalSplitDiffWorker struct {
 // NewVerticalSplitDiffWorker returns a new VerticalSplitDiffWorker object.
 func NewVerticalSplitDiffWorker(wr *wrangler.Wrangler, cell, keyspace, shard string, minHealthyRdonlyTablets, parallelDiffsCount int) Worker {
 	return &VerticalSplitDiffWorker{
-		StatusWorker:            NewStatusWorker(),
-		wr:                      wr,
-		cell:                    cell,
-		keyspace:                keyspace,
-		shard:                   shard,
+		StatusWorker: NewStatusWorker(),
+		wr:           wr,
+		cell:         cell,
+		keyspace:     keyspace,
+		shard:        shard,
```

Comment:
I didn't know that this passes the linter now. It seems to be a recent change?

My local goimports does the same thing, so that's fine with me :)